glossary.md

```diff
@@ -0,0 +1,43 @@
+# Glossary of Terms
+
+OpenAttestation: The underlying technology/framework that we use to power verifiable credentials
+
+OpenCerts: The project that is owned by SSG and is concerned with educational credentials
+
+TradeTrust: The project that is owned by IMDA and is concerned with trade documents as well as transferable asset deeds
+
+### OpenAttestation Framework
+  
+  Envelope: Wrapper around a document that uses OpenAttestation framework to protect its integrity. _(Sometimes called Signed JSON file)_
+  
+  Proof: Part of the `Envelope` metadata, a set of Keccak256 hashes that leads to the Merkle root that is notarised (i.e on the Ethereum blockchain)
+  
+  Document: The information inside the .opencert or .tt file that excludes the `Envelope` metadata. _(Sometimes called Unsigned JSON)_
+  
+  Schema: Refers to a specific schema that the contained `Document` has to adhere to.
+  
+  Template: Instructions on how to display the data in the `Document`, could be a URL pointing to a `Renderer` or simply a path depending on OpenAttestation version
```

Comment:
```suggestion
  Template: Instructions on how to display the data in the `Document`, it is currently an URL pointing to a `Renderer`. For OpenCerts, a simple path is used for v1.4 & v1.5, prior to that, a key is used.  
```

Reply:
We have two legacy in OpenCerts. 
Also wanted to show that going with the path/key is **not the norm**